pycape/llms/crypto.py

```diff
@@ -0,0 +1,45 @@
+import json
+import os
+from typing import Any
+from typing import Dict
+
+from cryptography.hazmat.primitives import hashes
+from cryptography.hazmat.primitives import serialization
+from cryptography.hazmat.primitives.asymmetric import padding
+from cryptography.hazmat.primitives.ciphers import aead
+
+NONCE_SIZE = 12
+
+
+def aes_decrypt(ctxt: bytes, data_key: bytes) -> bytes:
```

Comment:
I would add a test file for this but it's not a huge deal.

Reply:
Yeah i can add some tests.